client/src/App.js

```diff
@@ -1,12 +1,16 @@
 import React, { Component } from "react"
 import { BrowserRouter as Router, Route } from "react-router-dom"
+import Results from "./Components/Results"
+
+import "./App.css"
 
 import HomePage from "./Components/HomePage"
 
 const App = () => (
   <Router>
     <React.Fragment>
       <Route exact path="/" component={HomePage} />
+      <Route exact path="/results" component={Results} />
     </React.Fragment>
```

Comment:
as discussed the results should render on the same page as the search bar , so there doesnt need to be different end points and different pages for these